py/embedded-server/java-runtime/build.gradle

```diff
@@ -27,6 +27,10 @@ dependencies {
         runtimeOnly project(':hotspot-impl')
     }
 
+    if (!hasProperty('excludeClockImpl')) {
```

Comment:
This was also missing, but the server impl explicitly assumes it is there in its jvm args.